hoot-core/src/main/cpp/hoot/core/io/NonIdRemappingHootApiDbWriter.cpp

```diff
@@ -22,38 +22,46 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015, 2017 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016, 2017 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
This seemed odd to me but it looks like a rename of a file? And then completely changed?!  I'm assuming you deleted a file and then created a different file and this is what `git` did to it.  Is that correct?